build/environment.yml

```diff
@@ -8,7 +8,8 @@ dependencies:
   - anaconda::python=3.6.2
   - anaconda::pyyaml=3.12
   - anaconda::requests=2.14.2
-  - bioconda::wkhtmltopdf=0.12.3
+  - conda-forge::cairo=1.14.6
+  - conda-forge::cffi=1.10.0
```

Comment:
Do you want to add the cairocffi install back now that we will stay Linux-specific?  The environment [doesn't require it to be specified explicitly](https://travis-ci.org/greenelab/manubot-rootstock/builds/283358546#L599) so this should be fine as is.

Reply:
Given the lack of windows binaries, I'm fine with allowing pip to do its thing here. There's lot's of dependencies that we don't explicitly pin. I'm still unsure of what the right balance is here... conda doesn't really have documentation for these sorts of decisions (: